tasks/main.yml

```diff
@@ -28,21 +28,18 @@
 
 - name: Include CAT I patches
   include: fix-cat1.yml
-  when: rhel7stig_cat1_patch
```

Comment:
These conditional so should not be removed. They are what allow remediation based in severity.